lib/cosmos/system/target.rb

```diff
@@ -131,11 +131,12 @@ def process_file(filename)
         when 'REQUIRE'
           usage = "#{keyword} <FILENAME>"
           parser.verify_num_parameters(1, 1, usage)
+          filename = File.join(@dir, 'lib', parameters[0])
```

Comment:
This filename is only correct for a target lib require (which is tried first).  It is not correct for the other line I will comment on.